paddle/fluid/operators/optimizers/dgc_momentum_op.h

```diff
@@ -38,6 +38,22 @@ class DGCMomentumKernel : public framework::OpKernel<T> {
     auto current_step_tensor = context.Input<framework::Tensor>("current_step");
     auto* current_step = current_step_tensor->data<T>();
 
+    // nranks
+    auto nranks_tensor = context.Input<framework::Tensor>("nranks");
+    const int nranks = static_cast<const int>(*nranks_tensor->data<float>());
+    PADDLE_ENFORCE_GT(nranks, 1, "DGC is not useful when num_trainers <= 1");
```

Comment:
PADDLE_ENFORCE_GT(nranks, 1, "DGC is not useful when num_trainers <= 1, input %d", nranks);

Reply:
Done.